.github/ISSUE_TEMPLATE/1_add-font.md

```diff
@@ -15,18 +15,21 @@ assignees: ''
 
 **Requirements:**
 
-I understand that Google Fonts will publish only fonts that matches [its requirements](https://googlefonts.github.io/gf-guide/), and I can confirm the project meets them (by ticking the cases, or putting x between the square brackets in text mode):
 
+Google Fonts will publish only fonts that match its requirements. Please familiarize yourself with the complete documentation in the [Google Fonts Guide](https://googlefonts.github.io/gf-guide/) (GF-Guide) and ensure your font project complies with them before submitting the font family. You can also use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template), which will help you create a repository that follows the needed structure and includes build requirements.
+
+By filling this issue, you can confirm the project meets the  requirements (by ticking the cases or putting x between the square brackets in text mode):
+
+- [ ] I read, agree, and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
 - [ ] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
 - [ ] There is no "Reserved Font Name" mention in the OFL license
-- [ ] Family name is unique according to namecheck.fontdata.com
+- [ ] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
 - [ ] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
 - [ ] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
 - [ ] The source files are available in the repository
-- [ ] I read, agree and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/onboarding.html)
-- [ ] I am the author of the font or the author agreed to this submission
-- [ ] I will maintain the repository
+- [ ] I am the author of the font, or the author agreed to this submission
+- [ ] I will respond for the additional job required on the font for publishing it and will maintain the repository
```

Comment:
This line could maybe be more clear, something like:

`I will maintain the repository and participate in the onboarding process`